docs/Diagnostics.md

```diff
@@ -0,0 +1,82 @@
+This document contains information about diagnostic tools and outputs used for debugging and diagnosing eBPF issues.
+
+--------------------
+
+- [WFP State](#wfp-state)
+- [bpftool](#bpftool)
+- [eBPF Diagnostic Traces](#ebpf-diagnostic-traces)
+  - [Trace Providers](#trace-providers)
+  - [Logman Trace Command](#logman-trace-command)
+  - [Decoding Traces](#decoding-traces)
+  - [Viewing Traces](#viewing-traces)
+
+--------------------
+
+## WFP State
+
+Use the command `netsh wfp show state` to produce a `wfpstate.xml`. This file shows the WFP state on the system, including all WFP `sublayer`, `callout`, and `filter` objects. This can be used to determine if eBPF objects are correctly configured or if there are other callout objects present which may interfere with eBPF behavior.
```

Comment:
```suggestion
Use the command `netsh wfp show state` to produce a `wfpstate.xml`. This file shows the WFP state on the system, including all WFP `sublayer`, `callout`, and `filter` objects. This can be used to determine if eBPF objects are correctly configured or if there are other callout objects present that may interfere with eBPF behavior.
```

Also please try to keep source lines relatively short (say 120 chars or less,per https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#formatting-notes) since github has a hard time diffing really long lines.